system/languages/en.php

```diff
@@ -50,6 +50,7 @@
 		"default-page" => "default page",
 		"default-language" => "default language",
 		"digest-interval" => "digest interval",
+		"default-new-page-type" => "new page type",
```

Comment:
Did you intentionally leave out "default" here? To mimic the Dutch version, perhaps "default type for new pages" might be better?

Reply:
Not intentionally, fixed